src/components/search-result/SearchResults.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import './SearchResults.css';
+import Film from './film/Film';
+
+class SearchResults extends Component {
+  numbers = [1, 3, 4, 5];
+
+  render() {
+    return (
+      <div className="container">
+        <div className="row">
+          {
+            this.numbers.map(value => <Film id={value} />)
```

Comment:
You forgot about `key` prop. It's required prop for mapped components. See more info here: https://reactjs.org/docs/lists-and-keys.html